### ISSUE TITLE
#BSO FUTURE PROPOSAL?

### ISSUE THREAD

**Issue by EngageV2** [oldschoolgg/oldschoolbot#542]
Remove Raid Items from all banks giving 50% GP to the owner of the item upon deletion
Remove all Raids items from boxes

Add Raids
Add Slayer(with superiors)
Add ability to create the BIS boots
Add Revs
Add a Leagues (optional or just make a new bot for it called LeagueBotOldschool? LBO?)
Winners or top %ers get a chance for 1 month special name color with Role/title of their achievement
Buff Masses/Duo’s 
Make dice chuck 1/30 chance instead of 1/10
Make sell value of all items 90% instead of current
Make it so you have to have x amount of playtime or total leve before you can do a raid(can skip with patron)
Add =fp (flower poker)
Add BSO specific mods (😁)
Advertise BSO on the official website and basically everywhere

PROFIT

**Comment by Lajnux** [comment]
A simple version of raids is already PRed, just waiting on Magna to confirm/merge/review it. There will most likley be some kind of requirments for the BSO raids. #506 

Slayer is being worked on for OSB and will be adapted to BSO as soon as it gets approved in OSB. #429 

Revenants can be merged early to BSO see #482 . Or they will come with slayer.
Leagues is something that been thought of but currently isn't priotized due to want all skills in the bot first and working before starting to add fun leagues where you can choose custom abilities that makes your character unique.

Corp masses currently using a good dynamic formula, Duos might be looked over at a later stage, probably still very close to in game speeds.

Money drains are good for the health of any economy so having 80% sell value is okay. same with 1/10 dice chuck.

more gambling games can be looked into and for real ideas about this make a seperate issue. 

the moderator thing I commented on previous issue. #468 , BSO also got news channel and being advertised moderately.

Requesting to close issue and suggesting to make a new issue for more gambling games.
